test/server.jl

```diff
@@ -0,0 +1,24 @@
+# find a good port
+conn = listenany(2228)
+close(conn[2])
+port = conn[1]
+
+@async lintserver(port)
+sleep(1) #let sever start
```

Comment:
The server was being terminated after Pkg.test was done (somehow). I changed it so that the tests now explicitly close the server.